### PR TITLE
[FLINK-36224][docs] Add the version mapping between pipeline connectors and flink

### PR DESCRIPTION
**Pull request by Thorne-coder**
Support the version mapping between pipeline connectors and flink

### CODE REVIEW

**Review by gtk96 — APPROVED** [review]
LGTM  cc @leonardBang @Jiabao-Sun

**Comment by Thorne-coder** [comment]
@leonardBang PTAL

**Comment by Thorne-coder** [comment]
@yuxiqian   PTAL

File: flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mysql-cdc/src/test/resources/docker/server/my.cnf
Comment (yuxiqian) [inline flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mysql-cdc/src/test/resources/docker/server/my.cnf:1]: Is this change relevant to this PR?

Comment (Thorne-coder) [reply flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mysql-cdc/src/test/resources/docker/server/my.cnf:1]: sorry，it doesn't belong to this PR

**Comment by Thorne-coder** [comment]
another [pr:[](https://github.com/apache/flink-cdc/pull/3598)]
